src/Util/ClassSourceManipulator.php

```diff
@@ -94,7 +94,8 @@ public function addEntityField(string $propertyName, array $columnOptions, array
         if ('array' === $typeHint) {
             $defaultValue = new Node\Expr\Array_([], ['kind' => Node\Expr\Array_::KIND_SHORT]);
         }
-        $this->addProperty($propertyName, $comments, $defaultValue);
+
+        $this->addProperty($propertyName, $comments, $defaultValue, $isTyped ? $typeHint : null);
```

Comment:
If a property is type hinted, we should probably automatically default the value to `null` or `[]` to avoid initialization errors.
e.g.
```
private ?string $someString = null;
```

See https://rushlow.dev/blog/php-74-typed-properties-bc-problems for more information on this.